src/test/java/org/folio/edge/sip2/repositories/PatronRepositoryTests.java

```diff
@@ -1,6 +1,7 @@
 package org.folio.edge.sip2.repositories;
 
 import static org.folio.edge.sip2.api.support.TestUtils.getJsonFromFile;
+import static org.folio.edge.sip2.api.support.TestUtils.getMockedSessionData;
```

Comment:
Eclipse warns that this import is not used. All uses are via the `TestUtils` import on line 34. We should remove this static import or change the usage to just `getMockedSessionData()` instead of `TestUtils.getMockedSessionData()` an then remove the `TestUtils` import on line 34, assuming there are no more direct references to `TestUtils`.